app/models/book.rb

```diff
@@ -91,6 +91,7 @@ class Book < ActiveRecord::Base
   default_scope { where(in_store: true) }
 
   CSV_COLUMNS = ["Book Name", "ASIN"]
+  LEVELS_CONVERT = YAML.load(File.read(File.expand_path('../../../db/levelsConvert.yml', __FILE__)))
```

Comment:
Line is too long. [100/80]<br>Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.